ios/RNSpokestack.swift

```diff
@@ -315,6 +389,12 @@ class RNSpokestack: RCTEventEmitter, SpokestackDelegate {
             resolvers[RNSpokestackPromise.start] = resolve
             rejecters[RNSpokestackPromise.start] = reject
             pipeline.start()
+        } else {
+            reject(
+                "not_initialized",
```

Comment:
these feel like keys that could be more strongly typed and connected with the js key name via an enum

Reply:
I dried this up with a function to reject a promise with a "not initialized" error.